Source/WebCore/platform/graphics/ComplexTextController.cpp

```diff
@@ -698,6 +709,13 @@ void ComplexTextController::adjustGlyphsAndAdvances()
         unsigned previousCharacterIndex = m_run.ltr() ? std::numeric_limits<unsigned>::min() : std::numeric_limits<unsigned>::max();
         bool isMonotonic = true;
 
+#if USE(CORE_TEXT)
+        // Measure all of the glyphs at once. We don't hold the result because
+        // the later individual lookup will retrieve it from the cache.
+        if (glyphCount > 1)
+            (void)font.boundsForGlyphs(glyphs);
```

Comment:
is there a reason why we c-cast it here? I think the function is not marked as nodiscard